app/src/src_nonfreenet/org/breezyweather/sources/china/ChinaResultConverter.kt

```diff
@@ -199,15 +198,16 @@ private fun getPrecipitationProbability(forecast: ChinaForecastDaily, index: Int
 }
 
 private fun getHourlyList(
-    publishDate: Date,
     location: Location,
     hourlyForecast: ChinaForecastHourly
 ): List<HourlyWrapper> {
     if (hourlyForecast.weather == null || hourlyForecast.weather.value.isNullOrEmpty()) return emptyList()
 
+    val hourlyListPubTime = hourlyForecast.temperature?.pubTime ?: return emptyList()
```

Comment:
I don't agree to return no hourly forecast if there is no `pubTime`. Either make it mandatory (not `null`) in the parsing, or make it fallback to current hour.

Reply:
Actually I refer to is this code in `getMinutelyList`.

https://github.com/breezy-weather/breezy-weather/blob/207bc1374e8f25d6a1099c153994826cfb39a504/app/src/src_nonfreenet/org/breezyweather/sources/china/ChinaResultConverter.kt#L241

But I agree with what you said. In this new commit, if `pubTime` is null, it will fallback to the current hour.